docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,45 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+If you have common configuration options you want to apply to all profiles, you have to do this manually per profile. See xref:../using/profiles.adoc#setting-global-config-profiles[Setting Global Configuration Options]
+
+The current profile being used is called the active profile.
+By default {project} commands will run against the active profile.
+You can run the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command to determine the active profile.
+
+The profile feature introduced a global `--profile` flag and xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
+
+The --profile global flag can be specified with any {project} command to run it against the specific profile bypassing the currently active profile.
+
+NOTE: Commands are run against the default profile if you are not using the `--profile` flag and have not set a different active profile.
+
+[[creating-profiles]]
 == Creating Profiles
 
-You can create a new instance of {project} with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
+You can create a new instance of {project} with xref:../command-ref/minishift_start.adoc#[`minishift --profile PROFILE_NAME start`] command.
 
-NOTE: The default profile is `minishift`. It will be present by-default and user does not need to create it specifically.
+NOTE: The default profile is *minishift*. It will be present by default and user does not need to create it specifically.
```

Comment:
No need to address the user, whether directly or indirectly: We also want one sentence per line. In the case of admonitions, I find that it's easier to quickly see and edit them when they're put in explicit blocks:

```adoc
[NOTE]
====
The default profile is *minishift*.
It will be present by default and does not need to be created.
====
```